cpp/src/link_prediction/similarity_impl.cuh

```diff
@@ -58,7 +59,62 @@ rmm::device_uvector<weight_t> similarity(
     //        max weight((u,a), (a,v)).
     //    Use these to compute weighted score
     //
```

Comment:
Is this comment still relevant?

Reply:
Yes. Noticed it earlier, wanted to address with other PR comments. Will be fixed in next commit.